csun-metalab/nomi
# NAM-368 -- UI: Login page should have floating labels (see description)

# Description
Added floating labels to the login page.
  
# Testing
1. Make sure you are able to tab through the form.
2. Ensure floating labels appear and are in the proper position.
3. Make sure the floating labels work on different screen resolutions.
